test/staking/happyPath.staking.test.ts

```diff
@@ -173,7 +177,9 @@ async function startDockerImage() {
 async function signNodeWithEve() {
   await sleep(30000);
   const cont = await docker.listContainers();
-  const newRunnCont = cont.filter((x) => x.Image.includes(dockerImageName))[0];
```

Comment:
why this change?

Reply:
Image I received did not contain `dockerImageName`

Reply:
leave it as it was before plz